### PR TITLE
cmd/charmcmd: Better yaml output for resources.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Review by rogpeppe — CHANGES_REQUESTED** [review]
LGTM but please add a test too, thanks.

**Comment by fabricematrat** [comment]
fixes: https://github.com/juju/charmstore-client/issues/134

File: cmd/charm/charmcmd/listresources_test.go [hunk 1]
```diff
@@ -100,3 +100,30 @@ someResource 0
...
+	c.Assert(err, qt.IsNil)
+	s.uploadResource(c, id, "someResource", "content")
+
+	err = s.client.Publish(id, []params.Channel{params.StableChannel}, map[string]int{
+		"someResource": 0,
+	})
+	c.Assert(err, qt.IsNil)
+
+	stdout, stderr, code := run(".", "list-resources", "~bob/wordpress", "--format=yaml")
+	c.Check(stdout, qt.Equals, `\- name: someResource
```
Comment (rogpeppe) [inline cmd/charm/charmcmd/listresources_test.go:119]: What's the backslash doing there?

**Review by rogpeppe — APPROVED** [review]
LGTM with one minor question, thanks!

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-client-merge

**Pull request merged**